_includes/footer.html

```diff
@@ -17,100 +15,9 @@
           <li>Copyright JS Foundation and other contributors, <a href="https://js.foundation">https://js.foundation/</a></li>
         </ul>
       </footer>
-    </div><!-- /.container -->
-
-  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600" rel="stylesheet">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap.native/2.0.15/bootstrap-native.min.js" defer></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.0/anchor.min.js" defer></script>
-    <!-- at the end of the BODY -->
-  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
-  <script type="text/javascript"> docsearch({
-    apiKey: '891b0e977d96c762a3821e0c00172ac9',
-    indexName: 'eslint',
-    inputSelector: '#eslint-search-input',
-    algoliaOptions: { 'facetFilters': [["tags:docs", "tags:blog"]] },
-    debug: false // Set debug to true if you want to inspect the dropdown
-  });
-  var htmlClassList = document.documentElement.classList;
-  htmlClassList.remove("no-js");
-  htmlClassList.add("js");
-  </script>
-  {% if page.homepage %}
-  <script src="{{ site.url }}/assets/js/image-lazy-loader.js" defer></script>
-  {% endif %}
-  <script>
-    document.addEventListener('DOMContentLoaded', function() {
-      anchors.add('.doc h2, .doc h3, .doc h4, .blog h2, .blog h3, .blog h4');
-      {% if page.homepage %}
-      var hasClass = function(element, className) {
-        return document.querySelector(element).classList.contains(className);
-      };
-      var createNode = function (html) {
-        var range = document.createRange();
-        range.selectNode(document.getElementsByTagName('div').item(0));
-        return range.createContextualFragment(html);
-      };
-      function shuffleArray(array) {
-        for (var i = array.length - 1; i > 0; i--) {
-          var j = Math.floor(Math.random() * (i + 1));
-          var temp = array[i];
-          array[i] = array[j];
-          array[j] = temp;
-        }
-        return array;
-      }
-
-      var logosNode = document.querySelector('.logos');
```

Comment:
This is left over from when we had the "Who uses ESLint?" logos on the home page. Since those were removed, this script doesn't actually do anything.